%%%%%%%%% PROJECT DESCRIPTION  -- 15 pages (including Prior NSF Support)

\required{Project Description}
\begin{center}
%\emph{Maximum of 15 pages}
\end{center}•
%The Project Description (including Results from Prior NSF Support, which is
%limited to five pages) may not exceed 15 pages. Visual materials, including charts,
%graphs, maps, photographs and other pictorial presentations are included in the
%15-page limitation. PIs be cautioned that the project description must
%be self-contained and that URLs that provide information related to the proposal
%should not be used. \\
%
%All proposals to NSF are reviewed utilizing the two merit review criteria,
%intellectual merit and broader impacts. \\
%
% The Project Description should provide a clear statement of the work 
% to be undertaken and must include: objectives for the period of the proposed 
% work and expected significance; relation to longer-term goals of the PI's 
% project; and relation to the present state of knowledge in the field, 
% to work in progress by the PI under other support and to work in progress 
% elsewhere.

\section{Genetic architecture of highland adaptation in Zea mays}

\subsection{Questions}
\begin{itemize}
\item What is the genetic architecture of highland adaptation?
\item How much of the genetic architecture is shared between Mexico and South America?
\item  How much of the genetic architecture is shared between maize and teosinte?
\end{itemize}

\subsection{QTL mapping of highland adaptation traits in high x low mapping populations from Mexico and S. America}
\begin{itemize}
\item Phenotyping at 4 sites: 
\begin{itemize}
\item low Mexico (RS)
\item high Mexico (RS) 
\item Missouri (SFG)
\item growth chamber (MBH)
\end{itemize}
\item 960 F2:3 families; 40 checks
\item population development (RS, SFG)
\item genotyping and mapping (JRI, SFG)
\item DNA extraction, genotypes (SFG)
\item Sequence parents (JRI)
\item Mapping (SFG)
\end{itemize}

\subsection{Admixture mapping in mex/parv hybrid zone}
\begin{itemize}
\item collection trip (MBH)
\item 500 individuals from Ahuacatitlan in Mexico (MBH, RS)
\item DNA extractions, genotyping (MBH)
\item phenotyping (MBH, RS)
\item mapping (MBH/GC)
\end{itemize}

\subsection{Phenotypes}
\begin{itemize}
\item macrohairs (SFG, RS, MBH)
\item flowering time (SFG, RS, MBH)
\item tassel morphology (SFG, RS, MBH)
\item plant height every 2 weeks \& at flowering (SFG, RS, MBH)
\item biomass (SFG, RS, MBH)
\item \# ears (mz), 50k weight (mz/teo), total seed weight (mz) (SFG, RS, MBH)
\item stem/plant color (SFG, RS, MBH)
\item germination depth, temp in greenhouse (F2:3 only; MBH)
\item roots (inquire with Bloom)
\item ionomics (Ivan via letter of support)
\end{itemize}

\section{Introgression, admixture, and adaptation}
\subsection{Questions}
\begin{itemize}
\item Are introgressed loci adaptive?
\item Does evidence of introgression and natural selection correspond to QTL?
\item Are highland QTL/loci widespread in highland climes?
\end{itemize}

\subsection{Introgression and Admixture}
\begin{itemize}
\item GBS of 30 inds x 10 pops x 2 subpspecies (mex \& maize) (JRI)
\item Popgen on maize/mexicana introgression (JRI, GC)
\item GBS Additional 5 admixed parv/mex populations (50 inds. each) (JRI)
\item Introgression and adaptation in additional admix pops (JRI, GC)
\end{itemize}

\subsection{Global analysis of highland haplotypes}
\begin{itemize}
\item Occurrence of highland haplotypes/QTL/SNPs in global pops (MBH)
\item 500 worldwide accessions GBS (MBH)
\item Case study in Chihuahua (ACJ)
\end{itemize}

\section{Functional characterization of QTL}

\subsection{Questions}
\begin{itemize}
\item What do QTL/selected loci/introgressed loci do?
\end{itemize}

\subsection{Fine map pigmentation}
\begin{itemize}
\item PT x T43 NIL population (RS)
\item GBS genotyping (MBH)
\end{itemize}

\subsection{Allelic series for QTL of interest}
\begin{itemize}
\item 10 parents:
\begin{itemize}
\item 4 parents F2:3
\item mexicana TIL18
\item Palomero Toluqueño
\item 2 lowland landraces
\item 2 highland landraces
\end{itemize}
\item Cross into 3 parents for phenotyping
\begin{itemize}
\item B73 (SFG)
\item T43 (MBH)
\item CML457 (RS)
\end{itemize}
\end{itemize}

\subsection{RNAseq}
\begin{itemize}
\item Time series analysis of plants in field (ACJ):
\item 15 Lines
\begin{itemize}
\item 4 F2:3 parents
\item 1 NIL chr4,
\item 1 each mex \& parv TIL
\item B73, CML457, T43, PT
\item highland/lowland landraces used in allelic series (MBH to inbreed)
\end{itemize}
\item 12 plants per line per environment (2 pools of 6)
\item 4 stages/tissues per plant
\item 2 environments (high/low fields)
\end{itemize}

%Projects to be budgeted and written up:
%1) QTL mapping in high x low populations
%	SFG, RS, JRI, MBH
%2) Admixture mapping
%MBH, GC, RS
%3) Introgression: mexicana and maize
%	JRI, GC
%4) Global highland haplotypes
%	MBH, ACJ
%5) Functional Characterization: Fine\item mapping QTL & Allelic Series
%	RS, MBH, SFG
%6) Functional Characterization: RNASeq
%	ACJ
%7) Broader Impacts: Student Exchange
%	JRI
%8) Broader Impacts: Phenotyping Workshop
%	SFG




\required{Broader Impacts}
\setcounter{section}{1}
% As in the project summary, broader impacts must be called out separately 
% in the project description.  You may be able to give more specific
% examples, or discuss how you've previously achieved these impacts.
% It should be similar, but not identical, to the Broader Impacts statement
% in the project summary

Also mention online presence?  twitter, online dissemination, etc.? slideshare, figshare, preprints, Haldane's sieve?

\subsection{Exchange Program} %JRI, MBH, RS
\begin{itemize}
\item 2 students per year
\item 3-6 month stint in lab
\item Training: next-gen analysis, crossing \& phenotyping, mapping
\end{itemize}

\subsection{Phenotyping workshop} %SFG
\begin{itemize}
\item Yearly workshop at UM
\item Participants pay to purchase handheld
\item 2 day workshop provides software, training on phenotyping in maize
\end{itemize}

\subsection{Software} %JRI and GC
\begin{itemize}
\item R code for novel analysis for admixed populations
\item Software \& pipelines on github
\end{itemize}

\subsection{Germplasm resources} %MBH, SFG and RS
\begin{itemize}
\item NIL population \& F2:3 submitted to stock center
\end{itemize}

\required{Results From Prior NSF Support}
% 5 pages or fewer of the 15 pages for entire description document.
% include results from NSF grants received in the past 5 years.
% If supported by more than one grant, choose the most relevant one.

% For each grant, include: 
%	(a) NSF award number, amount, dates of support 
%	(b) The title of this project
%	(c) Publications resulting from this research
%	(d) Summary of the results of the completed work
%	(e) A brief description of data samples available and other research products not described 	      elsewhere
%	(f) For renewed support, a description of the relationship between the completed and 			      proposed work

% Due to space limitations, it is often advisable to use citations rather
% than putting the titles of the publications in the body 
% of this section

\subsubsection*{Ross-Ibarra: \#0922703: Functional Genomics of Maize Centromeres}
\emph{Intellectual merit} Centromeres are regions of the genome that organize and regulate chromosome movement, yet the biology of centromeres remains poorly understood. Co-PI Ross-Ibarra's group has focused in particular on the evolutionary genetics of centromeres. This work has demonstrated the remarkable evolutionary lability of centromere tandem repeats, but has shown that there is little evidence in maize for coevolution between centromere sequence and kinetochore proteins. Ongoing work from the Ross-Ibarra lab seeks to characterize kinetochore proteins, assess the phylogenetic evidence for longer-term coevolution, and understand patterns of centromere and genome size variation in natural populations. \\\emph{Broader impacts} Co-PI Ross-Ibarra has established and runs an international student exchange program as part of this grant. Data and result of this project have been disseminated via publications and presentations as well as deposited in the maize genetics community database \url{www.maizegdb.org}. Former trainees on the grant include Dr. Matthew Hufford (Co-PI on the current grant).
 
\begin{itemize}  \itemsep0em
\item Pyh\"aj\"arvi T, Hufford MB, Mezmouk S, Ross-Ibarra J (2013) Genome Biol \& Evol:  5:1594-1609
\item Hufford MB, Lubinsky P, Pyh\"aj\"arvi T, et al. PLoS Genetics 9(5): e1003477.
\item Melters DP, Bradnam KR, Young HA, et al. (2013) Genome Biology 14:R10
\item Kanizay LB, Pyhäjärvi T, Lowry E, et al. Heredity 110: 570-577.
\item Hufford MB, Bilinski P, Pyh\"aj\"arvi T, Ross-Ibarra J (2012) Trends in Genetics 12:606-615
\item Hufford MB, Xun X, van Heerwaarden J, et al. (2012) Nature Genetics 44:808-811
\item Chia J-M, Song C, Bradbury P, et al. (2012) Nature Genetics 44:803-807
\item Fang Z, Pyhajarvi T, Weber AL, et al. (2012) Genetics 191:883-894
\item Shi J, Wolf S, Burke J, Presting G, et al.  (2010) PLoS Biology 8: e1000327
\end{itemize}

\subsubsection*{Coop: \#1262645: Collaborative Research: ABI Innovation: Visualization And Statistics For Spatial Population Genomic Analysis. }
\$314,260, with an effective date of 05/01/13. Award Duration: 36 months.

\emph{Intellectual merit} We are developing a set of spatial statistics methods based on Gaussian random fields for the analysis of geographic population genomics data. The first method based on this approach has just been published (Bradburd et al. 2013), allowing a sound statistical framework to distinguish the effects of geographic and ecological distance on genetic isolation. \\
\emph{Broader impacts} The R package of the software has been released: \url{http://genescape.ucdavis.edu/scripts-and-code/bedassle/}, and has already been used by a many molecular ecologists. 

\begin{itemize} 
\item Bradburd, G., Ralph, P., Coop, G.  (2013) Evolution 67: 3258-3273
\end{itemize}

\subsubsection*{Flint-Garcia: \#0820619: Genetic Architecture of Maize and Teosinte}
\$ 9,823,000. 3/1/2009-2/28/2013. PI Edward Buckler, co-PIs J. Doebley, T. Fulton, S. Flint-Garcia, J. Holland, S. Kresovich, M. McMullen, Qi Sun. 

\emph{Intellectual merit} This project extends over more than a decade, and has pioneered the characterization of population genetic and evolutionary parameters of maize diversity, developed resources to connect this genetic diversity to phenotype through both association and joint linkage-association mapping, conducted fine scale analysis of domestication and agronomic QTL, and recently expanded to whole-genome analysis of diversity, evolution, and phenotype. Overall, the maize diversity project has developed a wide range of approaches and broadened understanding of the maize genome, evolution and adaptation, genetic mapping, and the agricultural improvement of maize. Over the last three years, the current iteration of the project (DBI-0820619) has successfully released and analyzed the maize Nested Association Mapping (NAM) population, collaborated on making first and second generation haplotype maps for maize, resolved domestication traits, developed a range of novel statistical approaches for association mapping, and dissected complex traits such as flowering time, kernel composition, disease resistance, height, and inflorescence and leaf morphology. Several positional cloning projects that will facilitate refinement of association mapping approaches are nearing completion.  We have also been collaborating with colleagues around the world to resequence more than 100 Zea genomes and skim sequence 20,000 maize varieties. Overall, the project has published 58 scientific papers in the last three years, including 15 in Science, Nature Genetics, and PNAS. \\
\emph{Broader impacts} The outreach program included a traveling science museum exhibit on maize diversity, evolution and genetics (seen by at least 300,000 people at five venues to date, including the famous Corn Palace in South Dakota), online Teacher Friendly Guide to the Evolution of Maize, seven Genotyping-By-Sequencing (GBS) workshops (held at primarily at Cornell but has also been held in Kenya), and training of postdocs, graduate students and undergraduates, the vast majority of which have continued in scientific careers.  Former trainees on this grant include Dr. Ross-Ibarra and Dr. Flint-Garcia (PIs of the current grant).  

\begin{itemize}  \itemsep0em\item Bottoms, C., Flint-Garcia, S. \& McMullen, M.D.  BMC Bioinformatics 11 Suppl 6, S28 (2010).
\item Brown, P.J. et al.  PLoS Genetics 7, e1002383 (2011).
\item Buckler, E.S. et al. Science 325, 714-8 (2009).
\item Chia, J.-M. et al. Nature Genetics 44: 803-807 (2012).
\item Cook, J.P. et al.  Plant Physiology 158, 824-834 (2012).
\item Dubois, P.G. et al. Plant Physiology 154, 173-86 (2010). 
\item Fang Z, Pyh\"aj\"arvi T, Weber AL, Dawe RK, Glaubitz JC, Sanchez Gonzalez JJ, Ross-Ibarra C, Doebley J, Morrell PL, Ross-Ibarra J. Genetics 191: 883-894. (2012)
\item Flint-Garcia, S.A., Bodnar, A.L. \& Scott, M.P.  Theoretical and Applied Genetics. 119, 1129-42 (2009).
\item Flint-Garcia, S.A., Buckler, E.S., Tiffin, P., Ersoz, E. \& Springer, N.M.  PLoS ONE 4, e7433 (2009).
\item Flint-Garcia, S.A., Dashiell, K.E., Prischmann, D.A., Bohn, M.O. \& Hibbard, B.E. Journal of Economic Entomology 102, 1317-1324 (2009).
\item Flint-Garcia, S., Guill, K. \& Sanchez-Villeda, H. Maydica 54, 375-386 (2009).
\item Hufford, M. et al. Nature Genetics 44: 808-811 (2012).
\item Hung, H.-Y. et al.  Heredity 1-10 (2011).
\item Hung H-Y, Shannon LM, Tian F, Bradbury PJ, Chen C, Flint-Garcia SA, McMullen MD, Ware D, Buckler ES, Doebley JF, \item Holland JB. PNAS 109: E1913-E1921. (2012)
\item McMullen, M.D. et al. Science 325, 737-40 (2009).
\item Morrell, P.L., Buckler, E.S. \& Ross-Ibarra, J. Nature Reviews Genetics 13, 85-96 (2011).
\item Ross-Ibarra, J., Tenaillon, M. \& Gaut, B.S. Genetics 181, 1399-413 (2009).
\item Studer, A., Zhao, Q., Ross-Ibarra, J. \& Doebley, J.  Nature Genetics 43, 1160-3 (2011).
\item Tian, F. et al.  Nature Genetics 43, 159-62 (2011).
\item van Heerwaarden, J. et al. Molecular Ecology 19, 1162-73 (2010).
\item van Heerwaarden, J., van Eeuwijk, F. a \& Ross-Ibarra, J.  Heredity 104, 28-39 (2010).
\item van Heerwaarden, J. et al. PNAS 108, 1088-1092 (2011).
\item Zhang, N, A Gur, Y Gibon, R Sulpice, S Flint-Garcia, MD McMullen, M Stitt, ES Buckler.  PLoS One 5: e9991 (2010)
\end{itemize}
