%%%%%%%%% PROJECT DESCRIPTION  -- 15 pages (including Prior NSF Support)

\required{Project Description}
\begin{center}
%\emph{Maximum of 15 pages}
\end{center}•
%The Project Description (including Results from Prior NSF Support, which is
%limited to five pages) may not exceed 15 pages. Visual materials, including charts,
%graphs, maps, photographs and other pictorial presentations are included in the
%15-page limitation. PIs be cautioned that the project description must
%be self-contained and that URLs that provide information related to the proposal
%should not be used. \\
%
%All proposals to NSF are reviewed utilizing the two merit review criteria,
%intellectual merit and broader impacts. \\
%
% The Project Description should provide a clear statement of the work 
% to be undertaken and must include: objectives for the period of the proposed 
% work and expected significance; relation to longer-term goals of the PI's 
% project; and relation to the present state of knowledge in the field, 
% to work in progress by the PI under other support and to work in progress 
% elsewhere.

\section*{Rationale and Significance}

%MBH: not sure we need this paragraph. if so, we should update it.

%Maize is a natural resource of fundamental national importance, vital for food, livestock feed, and fuel production.  Maize is by far the most valuable agricultural crop in the United States: in 2010, annual production in the U.S. was worth more than $66 billion dollars (USDA 2011), or ~60\% of the value of domestic crude oil production in 2009 (EIA 2010) and nearly double the value of soybean, the second most important field crop ($39 billion, USDA 2011). With a growing human population and an increased demand for alternative fuels, maize production must keep pace. Maize yield has steadily increased over the last three decades, but continued efforts are necessary to maintain or improve this yield trajectory.  An increase in the acreage planted to maize may accommodate some of this need, but it is clear that this is not a feasible long-term solution: from 1999-2009, for example, only ~30\% of the total increase in yield can be attributed to increased acreage (USDA 2009), and every day more arable land is lost to development. Some gains in yield may of course be won by improved farming practices, but the majority of yield increase can be directly attributed to breeding efforts (Duvick 1992, 2005), and breeding must remain of central importance in order to meet increased yield demands.

%MBH: We should decide if we're going with a climate change or "adaptation" theme.  I favor the latter. our project will help understand how maize adapts to new conditions -- important for breeding lines for highland regions, dry environments, or for understanding how maize might adapt to climate change.  will also be informative about how plants in general adapt since maize is so widespread and adaptable.  what do you think about this line?

%Climate change
%Changing climatic conditions pose a serious threat to our ability to continue increasing maize yield.  Historical analyses suggests that climate change over the last 30 years has already dramatically impacted maize yields worldwide, retarding the gains from breeding and management (Lobell et al. 2011a).  While these losses have been comparatively mild in the temperate climate of the United States, the same models suggest that change of even 1ºC could negatively impact yields by as much as 5-17\% (Lobell et al. 2003, Lobell et al. 2011a). Some models even predict that U.S. maize yields could be 30-46\% below current levels by the end of the century (Schlenker and Roberts 2009).  Given predictions for long-term changes in mean temperature (Fig. 1 A-C), it is reasonable to conclude that the impacts of climate change on maize yield could translate into economic losses of billions of dollars.  Moreover, these estimates do not take into account effects of the likely yield loss in the rest of the world (Lobell et al. 2011b) on prices and supply. Substantial efforts will be needed to preserve US maize production and increase or maintain yields, and in particular breeding efforts to adapt US maize to a warming climate must be a priority (Troyer 2004).  
While future climates will undoubtedly differ in a number of important ways, we focus in this proposal on adaptation to changing temperature.  Neither of the other major changes predicted as a result of climate change — elevated CO2 and changes in rainfall — are likely to be as important for maize yields.  Though elevated CO2 may actually prove beneficial for yields of some crops, models suggest the effect on maize, a C4 plant, to be relatively minor (Lobell et al. 2011a).  Similarly, analyses of past climate change find significant negative effects of temperature on maize yield but much weaker effects of precipitation (Lobell et al. 2003, Roberts and Schlenker 2009, 2010).  Increasing temperature, on the other hand, is likely to have strong, nonlinear effects on yield (Lobell et al. 2011b, Schlenker and Roberts 2009), as temperature extremes have an inordinate impact on overall yield.  
Improved breeding methods
Novel approaches to molecular plant breeding have been proposed as a solution to changing climes (Takeda and Matsuoka 2008), and some models suggest that breeding or other technological advances could meaningfully mitigate yield loss (Li et al. 2011).  Both marker-assisted selection (MAS) and transgenic approaches have become important tools of modern plant breeding, allowing breeders to combine traits of interest without the need for additional, costly phenotyping, pedigree analysis, or (in the case of MAS) a detailed functional understanding of the molecular basis of a trait. The success of both approaches, however, depend on our ability to identify useful markers and alleles.  Genetic mapping has been effective at identifying markers for use in MAS and candidates for transgenic methods.  But traditional mapping approaches such as quantitative trait locus (QTL) and association mapping have a number of drawbacks that may limit their utility or generality (see below). 
Selection mapping can circumvent some of these problems by utilizing changes in allele frequency to identify markers that have been, or are tightly linked to, the target of historical selection. We propose to extend our previous work on selection mapping for yield to identify candidate agronomic loci (CAL) that will prove useful for molecular breeding approaches to adapt maize to changing climates. Our approach takes advantage of several millennia of adaptation of traditional maize varieties to different climate regimes, natural experiments of grand scale, using selection mapping methods to identify the targets of selection in the independent adaptation of maize to highland environments in both Mexico and South America.  Importantly, differences between highland and lowland environments mirror the predicted effects of future climate change in the US (Fig. 1 C-E).  The resulting list of CAL, along with the bioinformatic tools we will develop, will provide new opportunities for molecular breeding, accelerating the progress of adaptation to climate change and thus ensuring continued yield increases.  Moreover, thanks to rapid advances in sequencing technology, the selection mapping approach proposed here should be easily extendible to virtually any crop species with sufficient germplasm resources; a modified method would be possible even for crops without a reference genome.  We therefore expect that our approach will prove to be an important advance in breeding methods for several crop species.

\subsection*{Maize diffusion and adaptation}
Despite its humble beginnings as a wild grass, maize spread rapidly across the globe to become the world’s top-producing crop.  Wild members of the genus Zea, which diverged relatively recently (Ross-Ibarra et al. 2009), are endemic to a small region stretching from northern Mexico to Central America. The direct wild ancestor of maize, Zea mays ssp. parviglumis, occurs only along the low-elevation slopes of the Sierra Madre Occidental in the southwestern corner of Mexico (Sánchez-González and Ruiz-Corral 1997).  After its domestication from ssp. parviglumis ~9,000 before present (BP; Matsuoka et al. 2002, Piperno et al. 2009), maize spread from the lowlands of Southwest Mexico (van Heerwaarden et al. 2011), rapidly diffusing across the Americas.  By ~6,000 BP, maize had adapted to the high elevations of central Mexico and spread to the lowlands of South America (Piperno 2006); by 4,000 BP maize was being grown at high altitudes in the Andes (Perry et al. 2006). After European contact with the New World, maize continued its world-wide diffusion, spreading quickly across Europe and subsequently to Asia and Africa.  Maize is now the world’s most broadly cultivated crop, currently grown on six continents in 162 countries and territories (FAOSTAT, 2009) in a distribution spanning 90° of latitude (from Chile to Canada) and more than 3000m of elevation (Tenaillon and Charcosset 2011).  During expansion to such varied regions, maize encountered — and adapted to — extremes of temperature, day length, precipitation, and soil types (Troyer 2004).  
There is clearly tremendous potential to harness the allelic variation present among maize populations for modern breeding, taking advantage of genotypes molded by selection to adapt cultivated maize to new and changing environments.  In particular, the fact that maize has independently adapted multiple times to similarly extreme environments (e.g., the highlands of the Mexican Central Plateau and the Andes) presents a unique opportunity to identify adaptive loci for use in maize breeding.  While these environments represent cooler, rather than warmer climates, identifying the loci underlying such adaptation in multiple populations should nonetheless be a powerful approach to finding CAL (see Approach below).

While variation in several well-known maize phenotypes (branching, glume architecture, endosperm color) is largely controlled by single loci (tb1, tga1, and y1 respectively), most complex traits have been found to be highly polygenic.  For instance, recent genome-wide association studies (GWAS) of leaf morphology, flowering time, and disease resistance have all found tens of QTL of small effect (Buckler et al. 2009, Tian et al. 2011, Kump et al. 2011).  These authors suggested that the outcrossing mating system of maize may explain the highly quantitative genetic architecture of adaptive traits in contrast to selfing species like rice and Arabidopsis where fewer loci are involved.  Adaptation to varying temperature during maize diffusion post-domestication is thus likely to have also involved a diffuse genetic architecture.  For example, tassel blasting and leaf firing, traits related to heat tolerance, have previously been shown to be polygenic in nature (Frova and Sari-Gorla 1994, Bai 2003) and temperature is known to affect many aspects of plant growth and development (e.g., seed germination, photosynthesis, respiration; Wahid et al. 2007).  

\subsection*{Preliminary results}

\begin{SCfigure}
  \centering
   \includegraphics[width=0.4\textwidth]{fst.pdf}
  \caption{Scatter plot of $-\log_10 empirical P$-values of $F_{ST}$ values in Mexico ($P_M$ on $x$-axes) and South America ($P_S$ on $y$-axes).   Red, blue, orange and gray dots represents SNPs showing significance in both Mexico and South America, only in Mexico, or only in South America, respectively.  The number of SNPs in each category is shown in the same color of dots.} 
\end{SCfigure}


blah blah F$_ST$ \ref{fig:fst}.

\renewcommand{\thesection}{Aim \arabic{section}}
\section{Genetic architecture of highland traits}

\subsection*{Questions}
\begin{itemize}
\item What is the genetic architecture of highland adaptation?
\item How much of the genetic architecture is shared between Mexico and South America?
\item How much of the genetic architecture is shared between maize and teosinte?
\item Are highland QTL/loci widespread in highland climes?
\end{itemize}

\subsection{QTL mapping of highland adaptation traits}
%\begin{itemize}
%\item Phenotyping at 4 sites: 
%\item 960 F2:3 families; 40 checks
%\item population development (RS, SFG)
%\item genotyping and mapping (JRI, SFG)
%\item DNA extraction, genotypes (SFG)
%\item Sequence parents (JRI)
%\item Mapping (SFG)
%\end{itemize}

In order to identify the genomic regions controlling highland adaptation in maize, we will conduct QTL mapping studies of one Mexican and one South American population, each derived by crossing an inbred landrace adapted to lowland conditions with a landrace adapted to highland conditions \ref{tab:qtlpops}.  We make use of specially-inbred landrace lines created by John Doebley (U. Wisconsin) and Seth Murray (Texas A\&M), thus simplifying downstream mapping applications and allowing replication of alleles in the our functional studies (see \ref{sec:funchar}). 

\begin{table}
\begin{center}
\caption{Parental lines for QTL} \label{tab:qtlpops}
\begin{tabular}{llll}\\\toprule  
{\bf Population}	& {\bf Parent } &	{\bf Origin (masl)} & {\bf Status }\\ \midrule
 \rowcolor{gray!25}
Mexico	& Zapalote Chico		& Oaxaca	 (46)		&  F2 \\ 
 \rowcolor{gray!25}
	& 	Palomero de Jalisco	& 	Jalisco (2520)		& \\
S. America	& Araguito	& Venezuela (183)	&  F1 \\ 
	& Sal Prieta	 & Ecuador (2948) & \\ \bottomrule
\end{tabular}
\end{center}
\end{table} 

Because we plan to evaluate these populations in replicated trials over multiple years, we will self-pollinate the F2 plants to create 500 F2:3 families from each population.  DNA will be extracted from each ofhte F2 plants and used for genotypic analysis.   The parents of the population will be sequenced to 20-30X depth on two lanes of Illumina 150bp paired-end, providing genome-scale SNP data similar in scale (tens millions of markers) to our previous work in maize  \citep[HapMap.v2;][]{Chia2012a}.  F2 plants will be genotyped using genotyping-by-sequencing \citep[GBS;][]{Elshire2011} and run through the standard maize GBS pipeline \citep{Glaubitz2014} resulting in approximately ~1M SNPs and allowing simple imputation of full-genome sequence.  The genetic map will be created using standard methods \citep[e.g.][]{Broman2003a}. 

\begin{table}
\rowcolors{2}{gray!25}{white}
\begin{center}
\caption{Parental lines for QTL} \label{tab:locales}
\begin{tabular}{p{2cm}cccccc}\\\toprule  
{\bf Field Sites} & {\bf Lat/Lon } & {\bf Elevation\,(m) } &	{\bf Min/Mean/Max\,$^{\circ}\mathrm{C}$  } & {\bf Precip\,(mm) } \\ \toprule
Valle de Banderas, Nayarit	& 20.8,\,-105.2&	54		&	15.3/25.8/33.7	&	1184 \\
Irapuato, Guanajuato 	&	20.7,\,-101.3	&	1729.0	&	7.3/20.2/31.7	&	693 \\
Amealco, Quer\'etaro 	&	19.5,\,-99.1	&	2240.0 	&	2.3/15.6/27.0	&	626\\
Columbia, Missouri		& 	28.9,\,-92.2	&	266.1 	&	-17.8/36.0/40.5&	914\\
Ames, Iowa	& 	X	&	X 	&	X		&	X	 \\ \bottomrule
\end{tabular}
\end{center}
\end{table} 

The populations will be phenotyped at 3 field locations, including one lowland site (Valle de Banderas in Mexico), one highland site (Irapuato or Queretaro, Mexico), and one temperate site near Columbia, Missouri (\ref{tab:locales}).  In each location, the experiment will consist of two replications where the 500 entries and parental checks will be arranged in a randomized complete block design or an augmented alpha lattice design. The parental checks will be used to control for field variation.  We will collect a large number of phenotypes (\ref{tab:phenos}) using our in-house barcode-based data collection program. Germination at different planting depths and temperatures will be evaluated in controlled conditions in Ames, Iowa, and root chilling will be evaluated using a custom hydroponic system at the University of California, Davis (see letter of support from Dr. Arnold Bloom).    

\rowcolors{2}{gray!25}{white}
\begin{wraptable}{r}{8cm}
\caption{Phenotypes measured}\label{phenos}
\begin{tabular}{ll}\\\toprule  
{\bf Abbreviation} & {\bf Phenotype}  \\\midrule
MH & leaf sheath macrohairs \\
FT & flowering time \\
PH & plant height \\
BM & total plant biomass \\
EN & ear number \\
FK & fifty kernel weight \\
TBN & tassel branch number \\
TL & tassel length \\
SM & total kernel mass \\
RC & root chilling response \\
GDP & germination depth \\
SC & stem anthocyanin content \\
GDP & germination depth \\
GDT & germination temperature \\\bottomrule
\end{tabular}
\end{wraptable} 

Raw data from each plot will be analyzed using mixed-models incorporating replications and environments.  Data will be analyzed across environments to determine whether location (elevation) affects the various phenotypes, as expected.  Each location will then be analyzed separately to derive least squares means to be used as the phenotypic data in QTL analyses.  QTL analysis will be conducted using standard, publicly available software (e.g. SAS; R/qtl \citealp{Broman2003a}).  Several iterations of QTL analysis will be conducted: on individual traits, individual traits adjusted for covariates such as flowering time, and multiple traits simultaneously.  The QTL profiles will be compared across populations (Mexican vs South American) and across field sites (elevations) to determine how elevation affects putatively adaptive traits.  We expect very different QTL profiles from the highland and lowland evaluation sites, and from the Mexican and South American populations.

%this needs something else. needs to talk about how we will answer questions above. that we will find how many QTL for each trait, that we expect this wil vary from fitness (yield) to color, thus informing us about the lability of these traits and how selection (or breeders) may act on them. also needs to mention fitness, no?  can we not use these pops to tell which traits are adaptive (i.e. genetically correlated to fitness/yield?)

\subsection{Admixture mapping in a teosinte hybrid zone}
%\begin{itemize}
%\item collection trip (MBH)
%\item 500 individuals from Ahuacatitlan in Mexico (MBH, RS)
%\item DNA extractions, genotyping (MBH)
%\item phenotyping (MBH, RS)
%\item mapping (MBH/GC)
%\end{itemize}


%\subsection{Phenotypes}
%\begin{itemize}
%\item macrohairs (SFG, RS, MBH)
%\item flowering time (SFG, RS, MBH)
%\item tassel morphology (SFG, RS, MBH)
%\item plant height every 2 weeks \& at flowering (SFG, RS, MBH)
%\item biomass (SFG, RS, MBH)
%\item \# ears (mz), 50k weight (mz/teo), total seed weight (mz) (SFG, RS, MBH)
%\item stem/plant color (SFG, RS, MBH)
%\item germination depth, temp in greenhouse (F2:3 only; MBH)
%\item roots (inquire with Bloom)
%\end{itemize}

\subsection{Global analysis of highland haplotypes}
\begin{itemize}
\item Occurrence of highland haplotypes/QTL/SNPs in global pops (MBH)
\item 500 worldwide accessions GBS (MBH)
\item Case study in Chihuahua (ACJ)
\end{itemize}

\section{Adaptive introgression of highland alleles}

\subsection*{Questions}
\begin{itemize}
\item Are introgressed loci adaptive?
\item Does evidence of introgression and natural selection correspond to QTL?
\end{itemize}

\subsection{Population genetics of maize-teosinte introgression} \label{subsec: intropopgen}

GBS of 18 inds x 10 pops x 2 subspecies (mex \& maize) (JRI)
384 at 48 plex * \$60 = \$23040

Popgen on maize/mexicana introgression (JRI, GC)
Identification of fine-scaled introgressed regions.
Evidence of selection against introgression (recombination)
Evidence of selection for introgressed regions (sweep signals, deviation of genome-wide admixture signal)
Introgressed regions correlate with admix mapping signals? with QTL? Berg approach

\subsection{Population genetics of hybridization in teosinte} \label{subsec: admixpopgen}

make use of admixed pop sampled by matt
Berg approach of phenotypes mapped there for selection in parentals and hybrid

GBS Additional 5 admixed parv/mex populations (50 inds. each) (JRI)
also 4 new pops x (12 parv + 12 mex + 12 hybrids) = \$8160
Ahuacatitlan and 3 more
Introgression and adaptation in additional admix pops (JRI, GC)
Selection for/against regions in parv/mex/admix
Parallelism across pops in hybrid zone

\section{Functional characterization of QTL} \label{sec:funchar}

\subsection{Questions}
\begin{itemize}
\item What do QTL/selected loci/introgressed loci do?
\end{itemize}

\subsection{Fine map pigmentation} \label{subsec:pigment}
\begin{itemize}
\item PT x T43 NIL population (RS)
\item GBS genotyping (MBH)
\end{itemize}

\subsection{Allelic series for QTL of interest} \label{subsec:series}
\begin{itemize}
\item 10 parents:
\begin{itemize}
\item 4 parents F2:3
\item mexicana TIL18
\item Palomero Toluqueño
\item 2 lowland landraces
\item 2 highland landraces
\end{itemize}
\item Cross into 3 parents for phenotyping
\begin{itemize}
\item B73 (SFG)
\item T43 (MBH)
\item CML457 (RS)
\end{itemize}
\end{itemize}

\subsection{RNAseq} \label{subsec:rnaseq}
\begin{itemize}
\item Time series analysis of plants in field (ACJ):
\item 15 Lines
\begin{itemize}
\item 4 F2:3 parents
\item 1 NIL chr4,
\item 1 each mex \& parv TIL
\item B73, CML457, T43, PT
\item highland/lowland landraces used in allelic series (MBH to inbreed)
\end{itemize}
\item 12 plants per line per environment (2 pools of 6)
\item 4 stages/tissues per plant
\item 2 environments (high/low fields)
\end{itemize}

%Projects to be budgeted and written up:
%1) QTL mapping in high x low populations
%	SFG, RS, JRI, MBH
%2) Admixture mapping
%MBH, GC, RS
%3) Introgression: mexicana and maize
%	JRI, GC
%4) Global highland haplotypes
%	MBH, ACJ
%5) Functional Characterization: Fine\item mapping QTL & Allelic Series
%	RS, MBH, SFG
%6) Functional Characterization: RNASeq
%	ACJ
%7) Broader Impacts: Student Exchange
%	JRI
%8) Broader Impacts: Phenotyping Workshop
%	SFG


\rowcolors{2}{gray!25}{white}
\begin{table}[H]
\begin{center}
\caption{Proposed timeline of activities and responsibilities}\label{timeline}
\begin{tabular}{lccccc}\\\toprule  
    \rowcolor{gray!50}
Year & 2015 & 2016 & 2017 & 2018 & 2019 \\\midrule
Objective \ref{subsec: series} Allelic series & 2015 & 2016 & 2017 & 2018 & 2019\\\midrule
Objective \ref{subsec: pigment} Fine mapping & -- & RS, AC & AC,JRI & AC,JRI & -- \\\midrule
Objective \ref{subsec: rnaseq} RNA-seq & 2015 & 2016 & 2017 & 2018 & 2019\\\midrule
Objective \ref{subsec: intropopgen} Maize/mexicana introgression & 2015 & 2016 & 2017 & 2018 & 2019 \\\midrule
Objective \ref{subsec: X} Admix mapping & 2015 & 2016 & 2017 & 2018 & 2019 \\\midrule
Objective \ref{subsec: X} QTL mapping & 2015 & 2016 & 2017 & 2018 & 2019 \\\midrule
Objective \ref{subsec: X} Highland haplotypes & 2015 & 2016 & 2017 & 2018 & 2019 \\\midrule
Objective \ref{subsec: admixpopgen} Admixture population genetics & 2015 & 2016 & 2017 & 2018 & 2019\\ \bottomrule
\end{tabular}
\end{center}
\end{table} 


\required{Broader Impacts}
% As in the project summary, broader impacts must be called out separately 
% in the project description.  You may be able to give more specific
% examples, or discuss how you've previously achieved these impacts.
% It should be similar, but not identical, to the Broader Impacts statement
% in the project summary

%Also mention online presence?  twitter, online dissemination, etc.? slideshare, figshare, preprints, Haldane's sieve?

\subsection*{Exchange Program} 

We propose an international student exchange program between the PIs in the U.S. and Senior Personnel at LANGEBIO in Mexico. Over the course of the grant, we propose to fund 10 graduate or undergraduate students for 3-month research internships in one of the collaborating laboratories. Students involved will participate in research projects directly relating to the research focus of the grant, including developing mapping populations, mapping traits, population genetic analysis, or analysis of next-generation data. The expectation is that such research will often lead to co-authorship on publications. Students will be asked to give two presentations, one to the host lab upon arrival, talking about the lab/university they came from and research there, and another to their host lab detailing their work over the 3-month period.  Each of the PIs will participate, sending students to Mexico and/or accepting students from Mexico for internships. PI Ross-Ibarra will manage the program, as he is fluent in Spanish and has past experience with a similar exchange program (NSF 0922703). Over the last four years, his lab has hosted 6 Mexican students who have worked on various computational aspects of centromere evolution. Two of those students have earned authorship on a paper to be submitted later this year and one has gone on to a PhD program in the U.S.Our goal is to involve students directly in research while at the same time fostering intercultural exchange and promoting future international research opportunities. It is particularly appropriate for the study of maize, a crop with significant cultural and economic impact in both Mexico and the U.S. Participating Mexican students will learn new analytical methods -- especially computational management of large datasets -- that can be introduced to their respective laboratories and peers. American exchange students will similarly benefit from experience with large field experiments and efforts to functionally characterize individual loci.  The hope is that Mexican undergraduate students involved may be recruited to graduate programs in the U.S., ideally to work in the lab of one of the PIs, and that American undergraduate students will be exposed to international opportunities for research, graduate education, and collaboration.

\subsection*{Phenotyping workshop} %SFG, JRI

The USDA-ARS group in Columbia has developed a streamlined phenotypic data collection system utilizing a handheld barcode device, barcoded plant tags, and barcoded phenotyping tools in order to maximize efficiency.  We will host a phenotyping workshop in Columbia during each year of the grant.  Through this workshop, Dr. Flint-Garcia’s state-of-the-art system will be transferred to other research institutions to aid in large-scale data collection. The phenotyping workshop will include topics on Experimental Design, setting up the FieldBook database, and Data Collection.  Experimental design topics include understanding where variation comes from, how to control for environmental/field variability and experimental error; heritability and repeatability.  The need for consistent data collection and high-throughput will be emphasized.  FieldBook database setup topics include setting up Palm handheld users, locations, traits, projects, assigning plots to projects, assigning traits and measurements to projects,  generating barcoded plant tags, and loading the program and trait groups to the Palm to prepare for data collection.  Topics to be covered in Data Collection include data collection for specific traits related to local adaptation of interest to our group, synchronizing data from the palm with the desktop/laptop database, managing data conflicts between the palm and the database, running reports, and exporting data.  This proposal will provide travel support for instructors.  The workshop will be free but participants will be expected to purchase their own Palm handheld and pay for their own travel.  The workshop will be held each year in late summer so that the participants can gain hands-on experience in data collection in the corn field.

\subsection*{Software} %JRI and GC
\begin{itemize}
\item R code for novel analysis for admixed populations
\item Software \& pipelines on github 
\end{itemize}

%needs tweaking/cutting, currently verbatim from Coop's last grant.

A good understanding of population and quantitative genetics is key to a student’s understanding of genetics and evolution, but these subjects are often conceptually quite difficult. An understanding of genetic variation and its phenotypic effects is also an increasingly important part of being an informed citizen, due to the rise of personal genomics and genomic medicine \citep[e.g.][]{redfield2012}. The large amount of population genetic and association data being generated offers a superb chance to motivate these subjects using real – and pertinent - data. 
We will develop undergraduate teaching modules in population and quantitative genetics using publicly available human data. These modules will be tested and integrated into the large undergraduate teaching courses at UC Davis. We have already begun to develop and distribute some of these resources, e.g. genome-scale demonstrations of Hardy Weinberg Equilibrium (HWE) using the human HapMap data, see Figure 4. Such demonstrations underscore the usefulness of basic population genetics in describing real world patterns, and begin to expose students to the wealth of genomics data being collected. They also offer the chance to more easily teach more complex population genetics concepts. For example, in a mixed population sample the heterozygosity is reduced below the HWE expectation by a factor (1-FST); this can easily be seen and explained for a combined sample of Europeans and Africans (right-side of Figure 4)
Other examples will include: using height association data to demonstrate quantitative genetics models; and explaining concepts of genetic and genealogical ancestry using genomic identity by descent. We will also develop software to examine signals of selection around human adaptations, to allow students to examine the patterns of diversity around these loci in the common visual framework. These modules will be prepared in the open source statistical program R, to ensure that they are easily used, modified, and distributed, and to expose students to programming in biology. The modules will be designed so that they can be tailored for use at a variety of levels from teaching basic concepts to large undergraduate classes, to providing the raw data for programming exercises for upper division courses.

The modules will be publicly distributed (see Data Management Plan), and advertised via evoldir and other venues. We will be develop and archive the modules on github.com in a fully open manner. The use of github will allow others to modify and extend the modules and to share and track these modifications. We will regularly deposit updated versions of the modules into figshare and data dryad in order to ensure that a permanent resource is maintained. 

\subsection*{Germplasm resources} %MBH, SFG and RS

%need emphasis on germplasm we will create (mapping pops etc.) that will be in germplasm banks

The highland environment remains an important niche for global maize production, in terms of acreage and farmer involvement, if not overall yield. The highland environment presents a number of important abiotic challenges beyond high elevation per se, including periods of drought and cold, and highland adapted material has great potential to provide an important source of stress tolerance; the rapid development and inherent earliness of highland material, for example, may have general application in marginal environments. 

In Mexico, the highland niche represents a key target market for an emerging private sector of small breeding companies, established following deregulation in 1990s. While highland adapted hybrids are available, these are largely derived from lowland sub-tropical material with little or no contribution of the highland landraces. This project will move this important germplasm a step closer to breeding programs, and in doing so strengthen ties between the broader academic community (represented here by both US and Mexican institutions) and the Mexican private sector (letter of support UNISEM).

This project will also build further the collaboration between US institutions and LANGEBIO, with a number of Mexican graduate students being directly involved. While Mexican plant science retains a traditionally strength in biochemistry and molecular biology, genetics has been less well represented in recent years. As a consequence, while there is ready access to genomics technologies, the human resources to make the most of the opportunities they present may be lacking. This project provides an excellent opportunity for capacity building in a Mexican institution, with the expectation of a lasting impact through future collaboration.

\required{Results From Prior NSF Support}
% 5 pages or fewer of the 15 pages for entire description document.
% include results from NSF grants received in the past 5 years.
% If supported by more than one grant, choose the most relevant one.

% For each grant, include: 
%	(a) NSF award number, amount, dates of support 
%	(b) The title of this project
%	(c) Publications resulting from this research
%	(d) Summary of the results of the completed work
%	(e) A brief description of data samples available and other research products not described 	      elsewhere
%	(f) For renewed support, a description of the relationship between the completed and 			      proposed work

% Due to space limitations, it is often advisable to use citations rather
% than putting the titles of the publications in the body 
% of this section

\subsubsection*{Ross-Ibarra, Flint-Garcia: \#1238014: Biology of Rare Alleles in Maize and Its Wild Relatives}
\$13,311,185 (\$2,368,767 to Ross-Ibarra and \$1,206,211 to Flint-Garcia), 05/15/13-04/30/18. PI Edward Buckler, co-PIs J. Doebley, J. Holland, S. Flint-Garcia, Q. Sun, P. Bradbury, S. Mitchell, J. Ross-Ibarra
\paragraph*{Intellectual merit} In the first year we have developed accurate imputation approaches, found evidence for the importance of deleterious variants and non-genic polymorphisms in heterosis and GWAS, documented differences in recombination among the parents of the NAM population, and found population genetic evidence suggesting the importance of demography and purifying selection across the genome.   The grant has produced 18 total publications in its first year (only publications involving PIs Flint-Garcia and Ross-Ibarra are shown below).
\paragraph*{Broader impacts} In the first year this project has included 10 postdoctoral and 12 graduate trainees. The GBS workshop and traveling maize exhibit continue to be popular and successful. A new version of the teacher-friendly guide to the evolution of maize has been revised and published online. 
\paragraph*{Publications} \citret{peiffer2013genetic, Romay2013, wills2013many, Mezmouk2014, Peiffer2014, sood2014mining}

\subsubsection*{Ross-Ibarra: \#0922703: Functional Genomics of Maize Centromeres}
\$5,008,031 (\$754,409 to Ross-Ibarra). 09/01/09-08/31/14. PI Kelly Dawe, co-PIs J. Birchler, J. Jiang, G. Presting, J. Birchler, J. Ross-Ibarra
\paragraph*{Intellectual merit} Centromeres are regions of the genome that organize and regulate chromosome movement, yet the biology of centromeres remains poorly understood. Co-PI Ross-Ibarra's group has focused in particular on the evolutionary genetics of centromeres. This work has demonstrated the remarkable evolutionary lability of centromere tandem repeats, but has shown that there is little evidence in maize for coevolution between centromere sequence and kinetochore proteins. Ongoing work from the Ross-Ibarra lab seeks to characterize kinetochore proteins, assess the phylogenetic evidence for longer-term coevolution, and understand patterns of centromere and genome size variation in natural populations. 
\paragraph*{Broader impacts} Co-PI Ross-Ibarra has established and currently runs an international student exchange program as part of this grant. Data and result of this project have been disseminated via publications and presentations as well as deposited in the maize genetics community database \url{www.maizegdb.org}. Former trainees on the grant include Dr. Matthew Hufford (Co-PI on the current grant).
\paragraph*{Publications} \citet{Shi2010a, Chia2012a, Fang2012a, Hufford2012, Hufford2012b, Hufford2013, Melters2013a, Kanizay2013, Pyhajarvi2013}

\subsubsection*{Coop: \#1262645: Collaborative Research: ABI Innovation: Visualization And Statistics For Spatial Population Genomic Analysis. }
\$314,260, with an effective date of 05/01/13. Award Duration: 36 months.
\paragraph*{Intellectual merit}  We are developing a set of spatial statistics methods based on Gaussian random fields for the analysis of geographic population genomics data. The first method based on this approach has just been published \citep{bradburd2013}, allowing a sound statistical framework to distinguish the effects of geographic and ecological distance on genetic isolation. \\
\paragraph*{Broader impacts} The R package of the software has been released online, and has already been used by many molecular ecologists. 

\subsubsection*{Flint-Garcia: \#0820619: Genetic Architecture of Maize and Teosinte}
\$ 9,823,000. 3/1/2009-2/28/2013. PI Edward Buckler, co-PIs J. Doebley, T. Fulton, S. Flint-Garcia, J. Holland, S. Kresovich, M. McMullen, Qi Sun. 
\paragraph*{Intellectual merit}  This project extends over more than a decade, and has pioneered the characterization of population genetic and evolutionary parameters of maize diversity, developed resources to connect this genetic diversity to phenotype through both association and joint linkage-association mapping, conducted fine scale analysis of domestication and agronomic QTL, and recently expanded to whole-genome analysis of diversity, evolution, and phenotype. Overall, the maize diversity project has developed a wide range of approaches and broadened understanding of the maize genome, evolution and adaptation, genetic mapping, and the agricultural improvement of maize. Over the last three years, the current iteration of the project (DBI-0820619) has successfully released and analyzed the maize Nested Association Mapping (NAM) population, collaborated on making first and second generation haplotype maps for maize, resolved domestication traits, developed a range of novel statistical approaches for association mapping, and dissected complex traits such as flowering time, kernel composition, disease resistance, height, and inflorescence and leaf morphology. Several positional cloning projects that will facilitate refinement of association mapping approaches are nearing completion.  We have also been collaborating with colleagues around the world to resequence more than 100 Zea genomes and skim sequence 20,000 maize varieties. Overall, the project has published 58 scientific papers in the last three years, including 15 in Science, Nature Genetics, and PNAS. \\
\paragraph*{Broader impacts} The outreach program included a traveling science museum exhibit on maize diversity, evolution and genetics (seen by at least 300,000 people at five venues to date, including the famous Corn Palace in South Dakota), online Teacher Friendly Guide to the Evolution of Maize, seven Genotyping-By-Sequencing (GBS) workshops (held at primarily at Cornell but has also been held in Kenya), and training of postdocs, graduate students and undergraduates, the vast majority of which have continued in scientific careers.  Former trainees on this grant include Dr. Flint-Garcia  and Dr. Ross-Ibarra  (PIs of the current grant), only their publications are shown below.
\paragraph*{Publications} \citet{Buckler2009, Flint-Garcia2009, Flint-Garcia2009a, Flint-Garcia2009b, Gore2009, McMullen2009, Ross-Ibarra2009a, Bottoms2010, Dubois2010, Zhang2010a, vanheerwaarden2010a, vanheerwaarden2010b, Brown2011b, Morrell2011a, Studer2011b, vanheerwaarden2011a, Tian2011, Chia2012a, Cook2012a, Fang2012a, Hufford2012b, Hung2012, Hung2012a, Romay2013}
